native/core/src/execution/datafusion/planner.rs

```diff
@@ -1290,6 +1285,21 @@ impl PhysicalPlanner {
         ))
     }
 
+    fn wrap_in_copy_exec(
+        is_sort_merge: bool,
+        plan: Arc<dyn ExecutionPlan>,
+    ) -> Arc<dyn ExecutionPlan> {
+        if is_sort_merge {
```

Comment:
Is there a more general description for SMJ's property to name this variable, and then we could use this function for the duplicated code elsewhere in this file?
https://github.com/apache/datafusion-comet/blob/31d0b0a38165e9315a2c0d981406dd48f7f7a3af/native/core/src/execution/datafusion/planner.rs#L920

Reply:
Yes, I will refactor this to reduce duplication